### DISCUSSION TITLE
Email reports plausible logo broken

### DISCUSSION THREAD

**Discussion by wheeleran** [plausible/analytics#3876] [Self-Hosted Support]
broken or blocked because/if the Plausible logo is being served from a remote resource? could this little svg/png not be served locally from our self-host? My weekly and monthly reports get sent SMTP via AWS to Gmail via MXroute mail server. i've Allowed remote resources in MXroute, the mails come through to Gmail without any Spam warnings or formatting errors in the report table itself, but the Plausible logo itself is a broken link. Or blocked. @ruslandoga is this something you could look at - if it's NOT me?

**Comment by ruslandoga** [comment]
👋 @wheeleran 

Sorry for a late reply. Yeah, the logo is broken in v2.0.0. It works in v2.1.0-rc.0.

**Reply by wheeleran** [reply]
that's the one i'm running, but i haven't received any reports since updating, weekly or monthly. haven't changed my docker file, was there something else i was supposed to do, with SMTP_HOST or GOOGLE_CLIENT?

**Reply by ruslandoga** [reply]
That's another problem that's fixed but the fix is not released yet. STARTTLS and TLS in general is broken in the SMTP client used in v2.1.0-rc.0. Final v2.1.0 or v2.1.0-rc.1 (if there is one) will include a new SMTP client https://github.com/plausible/analytics/pull/3654

**Reply by wheeleran** [reply]
oh dear. ok, it's clear, i'll wait. thanks again, as always.

**Reply by wheeleran** [reply]
@ruslandoga could you also add to the list a proper manifest site logo so that when the Plausible site is pinned to the home screen in mobile on Android we get a pretty little Plausible logo and not a blank placeholder? i believe this cropped up in an iOS thread a few years ago, but this hasn't been gotten to apparently, at least on Android?

**Reply by ruslandoga** [reply]
v2.1.0-rc.1 with the SMTP STARTTLS fix has beed released: https://github.com/plausible/analytics/discussions/4084

**Reply by wheeleran** [reply]
Does this release include a proper logo for pinned sites on Android home screens as well?

**Reply by ruslandoga** [reply]
I don't know. Android would probably just use the favicon.

**Reply by wheeleran** [reply]
But it's not pulling it, possibly because it's remote hosted? Any other site works, but not Plausible. There was a thread a few years ago about this for iOS too.

**Reply by wheeleran** [reply]
@ruslandoga hi, back to the issue of the Android 'add to home screen' missing the Plausible logo: is it something to do with [this](https://stackoverflow.com/questions/21018750/add-to-homescreen-button-in-android-does-not-show-website-as-a-web-app)? If it's been solved (?) for ioS, could the same be done for Android? The request has been popping up for [years](https://github.com/plausible/analytics/discussions/181).

**Reply by ruslandoga** [reply]
I was able to add my instance at https://who.edify.space to both Android and iOS. Favicon is used as the logo in both.

**Reply by wheeleran** [reply]
You're quiet right, I had to re-pin my shortcut, but there it is! Is there any way you could consider losing the CE black badge and actually centering the logo, the actual favicon?
